src/DynamoCore/Models/PortModel.cs

```diff
@@ -169,12 +135,7 @@ public bool DefaultValueEnabled
         /// </summary>
         public Thickness MarginThickness
```

Comment:
Another unfortunate leakage of UI concept into the model layer, but that's another story for another day.